shell/common/gin_converters/usb_protected_classes_converter.h

```diff
@@ -31,15 +33,14 @@ struct Converter<electron::ElectronPermissionManager::USBProtectedClasses> {
   static v8::Local<v8::Value> ToV8(
       v8::Isolate* isolate,
       const electron::ElectronPermissionManager::USBProtectedClasses& classes) {
-    std::vector<std::string> class_strings;
-    class_strings.reserve(std::size(classes));
+    absl::InlinedVector<std::string_view, ClassMapping.size()> class_strings;
```

Comment:
Q: how are you finding callsites that could use stack allocation ?

From https://groups.google.com/a/chromium.org/g/cxx/c/jTfqVfU-Ka0/m/caaal90NCgAJ the absl variant will also use heap allocation when the storage size is insufficient and there is also a trade-off in generated binary size.

Reply:
Yes, InlinedVector is best used when the container is temporary and its max size is small and known at compile time. These three things are all true for `class_strings`, since `ClassMapping` has only 7 elements in it.